src/kfactory/kcell.py

```diff
@@ -163,35 +164,42 @@ class KCellSettings(BaseModel, extra="allow", validate_assignment=True, frozen=T
     @model_validator(mode="before")
     def restrict_types(
         cls, data: dict[str, Any]
-    ) -> dict[str, int | float | SerializableShape | str]:
+    ) -> dict[str, int | float | SerializableShape | Sequence | str]:
```

Comment:
Ah, now I know what the problem is, the type should be:

`Sequence[int|float|SerializableShape|str]`

We could also leave the SerializableShape away, as I don't see much use case for that.

Reply:
do we want to allow bool as well?

Reply:
Yes, it's supported by klayout, so we can add it.

Reply:
When we do this 

`Sequence[int|float|SerializableShape|str]` i get pytest errors

and if we don't do it we get mypy errors :)

Reply:
works locally on 3.12 for me and in cicd. Probably time for updating pytest and mypy with `python -m pip install -e .[dev] -U --upgrade-strategy eager` ;)